tests/proofs/simple-arithmetic-spec.k

```diff
@@ -5,15 +5,17 @@ module SIMPLE-ARITHMETIC-SPEC
 
     rule <k> ( ITYPE:IValType . const X:Int ) => . ... </k>
          <stack> S:Stack => < ITYPE > X : S </stack>
-      requires 0 <=Int X
+      requires #inUnsignedRange(ITYPE, X)
 
     rule <k> ( ITYPE:IValType . const X:Int ) => . ... </k>
          <stack> S:Stack => < ITYPE > (X +Int #pow(ITYPE)) : S </stack>
-      requires X <Int 0
+      requires (#minSigned(ITYPE) <=Int X) andBool (X <Int 0)
 
     rule <k> ( ITYPE:IValType . const X:Int ) ( ITYPE . const Y:Int ) => . ... </k>
          <stack> S:Stack => < ITYPE > Y : < ITYPE > X : S </stack>
-      requires 0 <=Int X andBool 0 <=Int Y
+      requires #inUnsignedRange(ITYPE, X)
+               andBool
+               #inUnsignedRange(ITYPE, Y)
```

Comment:
Format like:

```
   requires #inUnsignedRange(ITYPE, X)
    andBool #inUnsignedRange(ITYPE, Y)
```

it scales better when you start having 20-30 `andBool`.

Reply:
Good to know, I think you mentioned it before. Done